miasm2/arch/x86/regs.py

```diff
@@ -342,6 +342,22 @@
 float_st6 = ExprId("float_st6", 64)
 float_st7 = ExprId("float_st7", 64)
 
+float_replace = {ExprId('ST', 64): float_st0,
```

Comment:
`ExprId('ST', 64)` is already declared at line 88, may a variable can avoid future error.